src/Config/RuntimeConfigLoader.cs

```diff
@@ -27,12 +23,11 @@ namespace Azure.DataApiBuilder.Config;
 /// which allows for mocking of the file system in tests, providing a way to run the test
 /// in isolation of other tests or the actual file system.
 /// </remarks>
-public class RuntimeConfigLoader
+public class RuntimeConfigLoader : BaseRunTimeConfigLoader
```

Comment:
```suggestion
public class FileSystemRuntimeConfigLoader : RuntimeConfigLoader
```

This would be dependent on the other change that I've suggested.

Reply:
Resolved.